internal/host/host.go

```diff
@@ -729,6 +732,34 @@ func (m *Manager) ReportValidationFailedMetrics(ctx context.Context, h *models.H
 	return nil
 }
 
+func (m *Manager) reportValidationStatusChanged(ctx context.Context, vc *validationContext, h *models.Host, newValidationRes map[string][]validationResult) error {
+	var currentValidationRes map[string][]validationResult
+	if h.ValidationsInfo != "" {
+		if err := json.Unmarshal([]byte(h.ValidationsInfo), &currentValidationRes); err != nil {
+			return errors.Wrapf(err, "Failed to unmarshal validations info from host %s in cluster %s", h.ID, h.ClusterID)
+		}
+		for vCategory, vRes := range currentValidationRes {
+			for i, v := range vRes {
+				// after reboot there is no agent, therefore, the host validation for 'connected' will constantly fail.
+				// this is the expected behaviour and we don't need to generate event/metric for it.
+				if v.ID == IsConnected && funk.Contains(manualRebootStages, h.Progress.CurrentStage) {
+					continue
+				}
+				if newValidationRes[vCategory][i].Status == ValidationFailure && v.Status == ValidationSuccess {
+					m.metricApi.HostValidationChanged(vc.cluster.OpenshiftVersion, h.ClusterID, vc.cluster.EmailDomain, models.HostValidationID(v.ID))
+					eventMsg := fmt.Sprintf("Host %v: validation '%v' that used to succeed is now failing", hostutil.GetHostnameForMsg(h), v.ID)
```

Comment:
Why is it %v and not %s?
Same comment to the other event.

Reply:
Fixed.